src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -4,69 +4,26 @@
   display: flex;
   position: relative;
 
-  .nav-search-icon {
-    align-self: center;
-    position: absolute;
-    left: 10px;
-    height: 20px;
-    width: 20px;
-    background-image: url(/icons/nav-link-search-icon.png);
-    background-size: 20px;
-    opacity: 0.4;
-  }
-
-  .nav-search-close-icon {
-    position: absolute;
-    align-self: center;
-    z-index: z(navbar-nav-search-close-icon);
+  .nav__clear-search {
```

Comment:
For the scss classes with names changed here, did we search across the entire codebase to ensure they aren't used/overridden anywhere else (and also update them there)?